src/App.tsx

```diff
@@ -1,61 +1,43 @@
+import { useState } from 'react';
+
 import './App.scss';
+import todosFromServer from './api/todos';
+
+import { getUserById } from './services/user';
+import { TodosWithUser } from './types/TodosWithUser';
+
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { TodoList } from './components/TodoList';
+
+export const todos: TodosWithUser[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
```

Comment:
Ensure that the `todosFromServer` data structure matches the expected `TodosWithUser` type. The `user` property should be correctly populated using the `getUserById` function.